glue/core/visual.py

```diff
@@ -94,6 +96,25 @@ def color(self, value):
         else:
             self._color = value
 
+    @callback_property
+    def preferred_cmap(self):
+        """
+        Color specified using Matplotlib notation
+
+        Specifically, it can be:
+
+         * A string with a common color (e.g. 'black', 'red', 'orange')
+         * A string containing a float in the rng [0:1] for a shade of
+           gray ('0.0' = black,'1.0' = white)
+         * A tuple of three floats in the rng [0:1] for (R, G, B)
+         * An HTML hexadecimal string (e.g. '#eeefff')
+        """
```

Comment:
This docstring seems out of date?

Reply:
Yup, it is. Thanks for pointing this out! Will update it shortly.